### PR TITLE
Fragment transformation

### PR DESCRIPTION
**Pull request by rachelbock**
Converted main activities to fragments and created a userData class and local database.
- Transformed all of the activities into fragments.
- Added the action bar directly to the Home Page activity and removed it's fragment.
- Added a userData class.
- Added a SQLiteHelper class to create a local databse to store the user data.

Rearrangement of project to include login activity. 
- Created a login activity to create and enter a user name.
- Created a local database to store user information from the user dialog fragment.
- Determined that it would be best to store data on external database.

Login Activity updated to be activity opened on launch of application. 
-Updated manifest to have Login Activity be main activity.
-On Submit Clicked buttons open home page.
-Got rid of writable database from Dialog Fragment. Will use External Database.

### CODE REVIEW

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java [hunk 1]
```diff
@@ -20,13 +22,14 @@
...
+    private UserSQLiteHelper userSQLiteHelper;
 
     @Bind(R.id.user_info_dialog_fragment_name_edit_text)
     EditText nameEditText;
-    @Bind(R.id.user_info_dialog_fragment_gender_edit_text)
-    EditText genderEditText;
-    @Bind(R.id.user_info_dialog_fragment_height_edit_text)
-    EditText heightEditText;
+    @Bind(R.id.user_info_dialog_fragment_height_ft_edit_text)
+    EditText heightFtEditText;
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java]: Go ahead and give all these fields an access modifier while you're in here

File: app/src/main/java/com/rage/clamber/Fragments/WallsFragment.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+import com.rage.clamber.R;
+import com.squareup.picasso.Picasso;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ *
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/WallsFragment.java:20]: Javadoc

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import com.rage.clamber.Adapters.ClimbsRecyclerViewAdapter;
+import com.rage.clamber.R;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * A simple {@link Fragment} subclass.
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java]: Add a bit of info about what this fragment's purpose is

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java (hunk 1 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java]: will do - I think those are ones that I had but then reset everything!

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java (hunk 3 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java]: will do

File: app/src/main/java/com/rage/clamber/Fragments/WallsFragment.java (hunk 2 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/WallsFragment.java:20]: Yep

**Pull request merged**